packages/backend/src/database/seeds/development/03_saved_dashboards.ts

```diff
@@ -72,5 +72,9 @@ export async function seed(knex: Knex): Promise<void> {
     await dashboardModel.create(spaceUuid, {
         name: 'Jaffle dashboard',
         tiles: [loomTile, markdownTile, ...chartTiles],
+        filters: {
```

Comment:
This should be optional

Reply:
I'm only adding support for 1 view in this PR.
More in this comment.
https://github.com/lightdash/lightdash/pull/1391#issuecomment-1041525876